common/test/acceptance/tests/helpers.py

```diff
@@ -267,7 +270,7 @@ def generate_course_key(org, number, run):
     return CourseLocator(org, number, run, deprecated=(default_store == 'draft'))
 
 
-def select_option_by_value(browser_query, value):
+def select_option_by_value(browser_query, value, focus_out=False):
```

Comment:
And moving the focus causes the test to fail? If you don't focus out, how is the onChange happening? It might be easier for us to chat tomorrow, as I don't think I understand the test.